### ISSUE TITLE
Beta site has several PageSpeed issues

### ISSUE THREAD

**Issue by gary-rowe** [bitcoin-solutions/multibit-website#203]
Worth providing a few simple fixes to gain a faster response overall.

See https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fbeta.multibit.org%2F&tab=desktop

**Comment by gary-rowe** [comment]
Optimised various aspects of the page in accordance with the Google guidelines. Not everything could be done so we'll just have to accept slightly laggy performance.

Ready for review and close.

**Comment by jim618** [comment]
I have built the site and checked there are no regressions.
I don't have any 'before and after' loading stats but it seems at least as good.
Closing

**Comment by jim618** [comment]
Noticed that the arrow to highlight the Accept button is now a missing glyph.
(Mac, Safari)

Screenshot:
![screen shot 2015-05-31 at 15 10 40](https://cloud.githubusercontent.com/assets/919427/7902172/5eaf9404-07a7-11e5-9b14-ff0430d710ee.png)

**Comment by jim618** [comment]
These mods also have broken:
+ Choose an operating system
+ Automatically get sent to download page
+ Go back to home page - all the download buttons have disappeared.

**Comment by jim618** [comment]
Note that I see this behaviour on Mac+Safari but not Mac+Chrome nor Mac+Firefox

**Comment by gary-rowe** [comment]
The arrow came from the glyphicon font which isn't really necessary and just slows things down. I've removed it and vertically centred the text leaving the screen to look as follows:

![screen shot 2015-05-31 at 15 45 40](https://cloud.githubusercontent.com/assets/342133/7902281/823b16c8-07ac-11e5-97c4-95c2a849616c.png)

After replicating your problems in Safari with no obvious reason why it's occurring, I've reinstated the original JS and CSS loading code in `head.ftl`. This immediately fixed the problem of the vanishing download section in Safari. We're not slow to download so it's not critical to make these changes right now.

Ready for review and close.
